doc/quick-reference-guide.md

```diff
@@ -0,0 +1,185 @@
+  ;; For Linux
+  input  (device-file "/dev/input/by-id/usb-04d9_daskeyboard-event-kbd")
+  output (uinput-sink "My KMonad output" "/run/current-system/sw/bin/sleep 1 && /run/current-system/sw/bin/setxkbmap -option compose:ralt")
+
+  cmp-seq ralt
+  cmp-seq-delay 5
+
+  fallthrough true
+
+  allow-cmd true
+)
+```
+
++ `defsrc`: define layer of input keys
+```
+(defsrc
+  grv  1    2    3    4    5    6    7    8    9    0    -    =    bspc
+  tab  q    w    e    r    t    y    u    i    o    p    [    ]    \
+  caps a    s    d    f    g    h    j    k    l    ;    '    ret
+  lsft z    x    c    v    b    n    m    ,    .    /    rsft
+  lctl lmet lalt           spc            ralt rmet cmp  rctl
+)
+```
+
++ `defalias`: define a button, can be referenced later using `@` in `deflayer`
+```
+(defalias
+  num  (layer-toggle numbers) ;; Bind num to a button that switches to a layer
+  kil  C-A-del                ;; Bind kil to a button that Ctrl+Alt+deletes
+)
+```
+
++ `deflayer`: define as many layers as you please (to toggle, add or remove).
+```
+(deflayer qwerty
+  grv  1    2    3    4    5    6    7    8    9    0    -    =    bspc
+  tab  q    w    e    r    t    y    u    i    o    p    [    ]    \
+  caps a    s    d    f    g    h    j    k    l    ;    '    ret
+  lsft z    x    c    v    b    n    m    ,    .    /    rsft
+  lctl @num lalt           spc            ralt rmet @sym @kil
+)
+
+(deflayer numbers
+  _    _    _    _    _    _    _    _    _    _    _    _    _    _
+  _    _    _    _    _    XX   /    7    8    9    -    _    _    _
+  _    _    _    _    _    XX   *    4    5    6    +    _    _
+  _    _    \(   \)   .    XX   0    1    2    3    _    _
+  _    _    _              _              _    _    _    _
+)
+```
+
++ `layer-toggle`: toggles to a specific layer as long as the key is held.
+```
+(defalias ked (layer-delay 500 delayed-layer))
+```
+
++ `layer-delay`: change the layer temporarily
+```
+(layer-delay 500 delayed-layer)
+```
++ `layer-next`: next key press is handled by another layer (leader-key style)
+```
+(defalias ken (layer-next next-layer))
+```
++ `layer-switch`: change the base layer
+```
+(defalias kes (layer-switch switch-layer))
+```
++ `layer-add`: overlays a layer on top of the current layer  
++ `layer-rem`: removes the overlayed layer from the current layer  
+**NOTE**: ***Using them is very dangerous, it is easy to make configurations which breaks the keyboard!***
+```
+(defalias
+  add (layer-add multi-overlay)
+  rem (layer-rem multi-overlay)
+)
+```
++ `around`: combines a bunch of keys into one key 
+```
+(defalias ad (around alt del)) ;; this is like pressing Alt+Del 
+```
+### Modded buttons
+`C-` : `(around lctl X)`  
+`A-` : `(around lalt X)`  
+`M-` : `(around lmet X)`  
+`S-` : `(around lsft X)`  
+Then `RC-`, `RA-`, `RM-`, and `RS-` behave exactly the same, except using the
+right-modifier.
+```
+(defalias Ca C-a) ;; this is equivalent to (defalias Ca (around Ctl a))
+```
```

Comment:
This seems a bit out of place here

Reply:
Why? I thought it would be helpful to add this part. I do not see why this should be removed.